<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>subDataWidget</class>
 <widget class="QWidget" name="subDataWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>28</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>40</width>
    <height>16</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>40</height>
   </size>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item>
    <widget class="QLabel" name="place1">
     <property name="text">
      <string notr="true">[]</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="place2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string notr="true">[]</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="place3">
     <property name="text">
      <string notr="true">[]</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="place4">
     <property name="text">
      <string notr="true">[]</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
